tests/unittest/_torch/auto_deploy/unit/singlegpu/models/test_bamba.py

```
@@ -0,0 +1,124 @@
+import torch  # noqa
```

Ruff (0.12.2)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)